tools/dockerfile-llb-frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.20 AS builder
```

Comment:
Btw, does buildkit use this `Dockerfile`, does it not require the `kraft` binary within? If so you can swap the `scratch` image with `kraftkit.sh/base:latest`

Reply:
This makes sense, and I can see the relevant code that does this. Thanks! :)